playbooks/example-source/group_vars/all

```diff
@@ -24,9 +24,8 @@ pulp_install_plugins:
   #   source_dir: "/var/lib/pulp/devel/pulp_npm"
   # pulp-python:
   #   source_dir: "/var/lib/pulp/devel/pulp_python"
-  # pulp-rpm:
-  #   source_dir: "/var/lib/pulp/devel/pulp_rpm"
-  #   prereq_role: "pulp.pulp_rpm_prerequisites" # RPM needs a prereq_role: https://github.com/pulp/pulp_rpm_prerequisites
+  pulp-rpm:
+    source_dir: "/var/lib/pulp/devel/pulp_rpm"
```

Comment:
So this playbook is actually meant for developers to run. It is also used in molecule CI. If we want molecule CI to use it, it should be copied to molecule/source-static/group_vars/all (and symlinked from other molecule tests) rather than symlinked into molecule folders.

Reply:
I see you created the new copies of the vars files under molecule/, but now you need to restore these example-use/ vars files to their original values (except for removing `prereq_role`).